bucket/everything-cli.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://www.voidtools.com/",
+    "description": "Command line interface to Everyting.",
+    "license": "MIT",
+    "version": "1.1.0.15",
+    "depends": "everything",
```

Comment:
```suggestion
    "depends": "extras/everything",
```

Reply:
OK, I didn't like the idea of depending on packages in other buckets, especially from main to extras, but if that's the way it's done...

Reply:
Should I just submit a new PR on main?

Reply:
Yeah, please close this one and submit a new PR with suggested change in Main.

Reply:
A quick search in main showed no other packages depending on packages in extras, only suggest's. I would say keep it in extras...

Reply:
I wonder if it is acceptable to add cli into everything.json and move it to main...

Reply:
Everything is a GUI tool, and does not fit the [criteria of main](https://github.com/lukesampson/scoop/wiki/Criteria-for-including-apps-in-the-main-bucket). I also don't think the cli fits the criteria, since it depends on a GUI tool.
I don't think the cli can be added to `everything.json`, as it has its own versioning.